pkg/kubelet/kubelet_test.go

```diff
@@ -564,7 +565,7 @@ func TestSyncPodsCreatesNetAndContainer(t *testing.T) {
 	waitGroup.Wait()
 
 	verifyCalls(t, fakeDocker, []string{
-		"list", "list", "list", "create", "start", "inspect_container", "create", "start", "list", "inspect_container", "inspect_container"})
+		"list", "list", "list", "inspect_image", "create", "start", "inspect_container", "create", "start", "list", "inspect_container", "inspect_container"})
```

Comment:
Before, if the pod infra container is not running and no user containers can be found, we'd return an error. Now, it'd continue to see if there is any creation error and inspect the images for the user containers.